src/TRestAxionOpticsMirror.cxx

```diff
@@ -334,15 +334,15 @@ Double_t TRestAxionOpticsMirror::GetReflectivity(const Double_t angle, const Dou
 
     Double_t ang = angle;
     if (ang < 0.0) {
-        warning << "Angle is below 0 degrees! It should be between 0 and 90 degrees" << endl;
+        warning << "Angle is below 0 degrees! It should be between 0 and 9 degrees" << endl;
         warning << "Setting angle to 0 degrees" << endl;
         ang = 0.0;
     }
 
-    if (ang > 90) {
-        warning << "Angle is above 90 degrees! It should be between 0 and 90 degrees" << endl;
-        warning << "Setting angle to 90 degrees" << endl;
-        ang = 90;
+    if (ang > 9) {
```

Comment:
Given the number of times this hardcoded number appears, it is one of the actual good use cases for a simple `#define NUM_ANGLES 9` thing (feel free to bikeshed the name).

Reply:
Right, I hope this value is not going to be changed much in the future, hope never, but yes, it will make the code more comprehensive.

Reply:
Actually I prefer to leave it as it is for the moment.